cmd/kusk/cmd/ip.go

```diff
@@ -111,16 +110,14 @@ var ipCmd = &cobra.Command{
 				}
 			}
 		}
-
 		//kubectl port-forward svc/kusk-gateway-dashboard -n kusk-system 8080:80
 		if svc.Spec.Type == "ClusterIP" {
-			err := fmt.Errorf("your envoyfleet doesn't have a Public IP address assigned. Try portforwarding %q", fmt.Sprintf("kubectl port-forward svc/%s  -n %s 8080:%d", svc.Name, svc.Namespace, svc.Spec.Ports[0].Port))
+			err := fmt.Errorf("your envoyfleet doesn't have public IP address assigned. Try portforwarding %q", fmt.Sprintf("kubectl port-forward svc/%s  -n %s 8080:%d", svc.Name, svc.Namespace, svc.Spec.Ports[0].Port))
```

Comment:
Please change to `doesn't have a`, and `Try port-forwarding`.

Reply:
done. Take another look pls